airgun/entities/host.py

```diff
@@ -243,26 +243,34 @@ def set_puppet_class_parameter_value(self, entity_name, name, value):
         view.flash.assert_no_error()
         view.flash.dismiss()
 
-    def get_webconsole_content(self, entity_name):
+    def get_webconsole_content(self, entity_name, use_new_ui=True):
         """Navigate to host's webconsole and return the hostname from the cockpit page
 
         :param str entity_name: The host name for which to set the parameter value.
+        :param bool use_new_ui: When true: use new UI elements for RHEL8 and RHEL9.
```

Comment:
I would maybe use int instead of bool, non-blocking

Reply:
Updated to use RHEL version instead of bool. This would be also easier to use from robottelo